apps/ui/src/components/Modal/DelegationConfig.vue

```diff
@@ -69,14 +64,15 @@ const definition = computed(() => {
                 'https://api.thegraph.com/subgraphs/name/arr00/uniswap-governance-v2'
               ]
             },
-            [networkField.value]: {
-              type: ['string', 'number', 'null'],
+            chainId: {
+              type: ['number', 'null'],
```

Comment:
This is failing when selecting Starknet as network

![Screenshot 2024-10-25 at 19 20 30](https://github.com/user-attachments/assets/c2920eb4-588b-428b-a719-5374209d5f8c)